titan-server/src/schema.rs

```diff
@@ -252,25 +278,20 @@ joinable!(wcf1_user_option_value -> wcf1_user_avatar (user_avatar_id));
 joinable!(wcf1_user_to_group -> wcf1_user (user_id));
 joinable!(wcf1_user_to_group -> wcf1_user_group (group_id));
 joinable!(wcf1_user_activity_event -> wcf1_user (user_id));
+joinable!(wcf1_user_group_option_value -> wcf1_user_group_option (option_id));
+joinable!(wcf1_user_group -> wcf1_user_group_option (group_id));
```

Comment:
I think this association is off. This statement implies `group_id` is foreign key in `wcf1_user_group`, which references `wcf1_user_group_option`, but these two tables don't have any common FK fields.

Based on the joins in the ACL query, I think this meant to be:

```rust
joinable!(wcf1_user_group_option_value -> wcf1_user_group (group_id));
```

Reply:
Yeah, you're right about the association being off. What I actually wanted is this:

```rust
...
joinable!(wcf1_user_group_option_value -> wcf1_user_group (group_id));
joinable!(wcf1_user_group_option_value -> wcf1_user_group_option (group_id));
```

I needed both because I'm joining across two different tables. This now works.